tools/cli/src/toolbox/registry/restart.ts

```diff
@@ -0,0 +1,34 @@
+// @/toolbox/registry/restart
+// View registry restart
+
+import { RegistryContainer, Toolbox } from '../../types'
+
+// Accept a toolbox, return a registry restart viewer
+export default async (toolbox: Toolbox) => {
+	const { events, system } = toolbox
+
+	events.emit('registry.restart', {
+		status: 'progress',
+		operation: 'restart-registry',
+		message: 'Restarting all registry containers',
+	})
+
+	await system
+		.exec('docker compose up --force-recreate -d')
```

Comment:
force recreate will destroy container, may not be expected behavior

Reply:
I use `docker compose up -d --force-recreate` to restart the containers after configuring keycloak and regenerating the  client secret in the `registry init` command as well. It ensures that the registry container picks up on the new client secret or any other configuration changes and doesn't cause a loss of configuration in the keycloak container either. 

On the other hand, `docker compose restart` did not get the registry to pick up the new/changed configuration, and that is the main purpose of the `registry restart` command.

Reply:
I have also added a `--soft` option to enable restarting without destroying the container.